### PR TITLE
List and allow editing of facts on main page

### LINKED ISSUE #10: Enable editing of facts on "main page"

### PR DESCRIPTION
**Pull request by pimotte**
Minimal logic to keep the facts the same type is in place, so this mildly relies on getting that right in the first place.

Closes #10

### CODE REVIEW

File: compliance-by-design-demo/src/components/FactChangeModal.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      value: event.target.value,
+    });
+  }
+
+  handleSave() {
+    console.log("Saving", this.props.factName, this.state.value);
+    if (this.props.handleSave) {
+      if (typeof this.props.oldValue === "number") {
+        this.props.handleSave(this.props.factName, Number(this.state.value));
+      } else {
```
Comment (hidde-jan) [inline compliance-by-design-demo/src/components/FactChangeModal.js:27]: Not yet implemented for strings?

Comment (pimotte) [reply compliance-by-design-demo/src/components/FactChangeModal.js:27]: `this.state.value` is a string, so in the case of a string no conversion is necessary

Comment (hidde-jan) [reply compliance-by-design-demo/src/components/FactChangeModal.js:27]: Yeah, but also nothing happens currently it seems

Comment (pimotte) [reply compliance-by-design-demo/src/components/FactChangeModal.js:27]: Gotcha. I forgot to retest the string case after testing the number case. Good catch:)

**Comment by hidde-jan** [comment]
The `else` branch is empty in you `handleSave`, but otherwise looks good 👍 I checked it out locally.

When we figure out how to neatly present the questions in the factprompt, we can re-abuse that here.

**Pull request merged**